SOURCES/MAKEFILE

```diff
@@ -50,8 +49,22 @@ OBJETS7  = msg_cust.obj
 
 EXE      = LBA0
 
-$(EXE).exe: $(OBJETS1) $(OBJETS2) $(OBJETS3) $(OBJETS4) $(OBJETS5) $(OBJETS6) $(OBJETS7)
-	echo "run LINK.bat under 4DOS"
+$(EXE).exe: LINK.LD
+	$(WL) Name $@ @ $<
```

Comment:
This translates to
```
wlink Name LBA0.exe @ LINK.LD
```
The `@` linker directive means "read more linker directives from this file"

Reply:
Changed in latest diff to having all linker directives passed to the linker without the need for a temporary file